payload-functions.js

```diff
@@ -259,6 +255,23 @@ function prepareAddSecurityGroupRulesPayload(params) {
   };
 }
 
+function parseSinglePortRange(rawPortRange) {
```

Comment:
`payload-functions.js` is a big file. This function could go into local `utils` file.